src/Lib2.hs

```diff
@@ -16,6 +16,12 @@ type Database = [(TableName, DataFrame)]
 
 -- Keep the type, modify constructors
 data ParsedStatement = ParsedStatement
+{
+  aggregateList :: [(Maybe String, String)],
+  tableName :: String,
```

Comment:
Maybe I'm wrong, but don't we need to support multiple tables? (e.g. "SHOW TABLES")

Reply:
For the "SHOW TABLES" we could maybe have a separate constructor, as in the instructions for the "SELECT" statement parsing supporting multiple tables  is not mentioned. 

For instance, 

> column list:
    Parse and recognize column names in a given query.
    Return the specified columns from the table in the result.
    Ensure provided column names exist in the table.

Table is always singular.